pkg/apmintegration/helpers.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package apmintegration
+
+import (
+	"errors"
+	"fmt"
+	"path/filepath"
+	"strings"
+)
+
+func getGithubOrg(url string) (string, error) {
+	split := strings.Split(url, "/")
+
+	if len(split) < 3 {
+		return "", errors.New("unable to find organization")
+	}
+
+	return split[len(split)-2], nil
+}
+
+func getGithubRepo(url string) (string, error) {
+	base := filepath.Base(url)
+	if base[len(base)-4:] != ".git" {
```

Comment:
The `path` package has the `Ext` function. https://pkg.go.dev/path#Ext

Reply:
Right now, we're requiring every URL end in `.git`, but I might try to remove that.

Reply:
I removed the requirement to end in `.git`. The tool now accepts with or without the extension.